_notices/rsn0024.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 24 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Deprecation announcement for Ubuntu 18.04 in v22.12"
+notice_author: RAPIDS Ops
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v22.12+"
+notice_created: 2022-11-21
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-11-21
+---
+
+## Overview
+
+Ubuntu 18.04 will be deprecated in our upcoming release of v22.12 scheduled for December 2022. After the v22.12 release, users should expect that support for CUDA 11.0 may be dropped at any point.
+
+
+## Impact
+
+Users should consider switching to any of the following supported operating systems for any `rapidsai/rapidsai` image:
+  - Ubuntu 20.04
+  - Ubuntu 22.04
+  - CentOS 7
+  - CentOS 8
```

Comment:
Should this be Rocky Linux 8? The tags say [cuda11.5-runtime-rockylinux8-py3.9](https://hub.docker.com/layers/rapidsai/rapidsai/cuda11.5-runtime-rockylinux8-py3.9/images/sha256-e6de9540a1b5be7c7968590cf8feee66da90989cbc09863cfc910037dce2fc96?context=explore).
```suggestion
  - Rocky Linux 8
```